payloads/node/scripts/test-run.sh

```diff
@@ -50,13 +50,14 @@ if [[ "$1" == "test" || "$1" == "test-all" ]]; then
 	${KM_BIN} ${TEST_KM} --gtest_filter="*"
 fi
 
-# if [[ "$1" == "test-all" ]]; then
-#    NODETOP=$5
-#    BUILD=$6
-# 	cd ${NODETOP}
-#    python tools/test.py -J --mode=`echo -n ${BUILD} | tr '[A-Z]' '[a-z]'` --skip-tests=`cat ../skip_* ../${PLATFORM_ID}_skip | tr -s '\n ' ','` default addons js-native-api node-api
-#    echo "Tests are Successful"
-# fi
+if [[ "$1" == "test-all" ]]; then
+   NODETOP=$5
+   BUILD=$6
+   echo -e "#!/bin/bash\n$(realpath ${KM_BIN}) $(realpath ${NODETOP}/out/${BUILD}/node.km) \$*\n" > /tmp/nodexxx && chmod +x /tmp/nodexxx
```

Comment:
Consider using mktemp to create a temp file instead of hardcoded /tmp/nodexxx.

Can you remove the temp file after the test is done?